code/modules/mob/living/carbon/xenomorph/abilities/predalien/predalien_abilities.dm

```diff
@@ -1,52 +1,90 @@
-/datum/action/xeno_action/activable/pounce/predalien
-	name = "Leap"
-	ability_primacy = XENO_PRIMARY_ACTION_1
-
-	knockdown = FALSE
-
-	distance = 5
-	knockdown = FALSE // Should we knock down the target?
-	slash = FALSE // Do we slash upon reception?
-	freeze_self = FALSE // Should we freeze ourselves after the lunge?
-	should_destroy_objects = TRUE   // Only used for ravager charge
-
 /datum/action/xeno_action/onclick/predalien_roar
 	name = "Roar"
 	action_icon_state = "screech"
 	ability_name = "roar"
 	action_type = XENO_ACTION_CLICK
 	ability_primacy = XENO_PRIMARY_ACTION_2
 	xeno_cooldown = 25 SECONDS
-	plasma_cost = 50
+	plasma_cost = 0
 
 	var/predalien_roar = list("sound/voice/predalien_roar.ogg")
 	var/bonus_damage_scale = 2.5
 	var/bonus_speed_scale = 0.05
 
-/datum/action/xeno_action/onclick/smash
-	name = "Smash"
-	action_icon_state = "stomp"
-	ability_name = "smash"
-	action_type = XENO_ACTION_CLICK
-	ability_primacy = XENO_PRIMARY_ACTION_3
-	xeno_cooldown = 20 SECONDS
-	plasma_cost = 80
 
-	var/freeze_duration = 1.5 SECONDS
 
-	var/activation_delay = 1 SECONDS
-	var/smash_sounds = list('sound/effects/alien_footstep_charge1.ogg', 'sound/effects/alien_footstep_charge2.ogg', 'sound/effects/alien_footstep_charge3.ogg')
 
-/datum/action/xeno_action/activable/devastate
-	name = "Devastate"
-	action_icon_state = "gut"
+/datum/action/xeno_action/activable/feralfrenzy
+	name = "Feral Frenzy"
+	action_icon_state = "rav_eviscerate"
 	ability_name = "devastate"
 	action_type = XENO_ACTION_CLICK
-	ability_primacy = XENO_PRIMARY_ACTION_4
-	xeno_cooldown = 20 SECONDS
-	plasma_cost = 110
+	ability_primacy = XENO_PRIMARY_ACTION_3
+	xeno_cooldown = 15 SECONDS
+	plasma_cost = 0
 
+	var/activation_delay_aoe = 1 SECONDS
+	var/base_damage_aoe = 15
+	var/damage_scale_aoe = 10
 	var/activation_delay = 1 SECONDS
-
 	var/base_damage = 25
 	var/damage_scale = 10 // How much it scales by every kill
+	var/targetting = SINGLETARGETGUT
+
+
+/datum/action/xeno_action/onclick/toggle_gut_targetting
+	name = "Toggle Gutting Type"
+	action_icon_state = "rav_scissor_cut" // default = heal
+	macro_path = ""
+	action_type = XENO_ACTION_ACTIVATE
+	ability_primacy = XENO_PRIMARY_ACTION_5
+
+/datum/action/xeno_action/onclick/toggle_gut_targetting/can_use_action()
+	var/mob/living/carbon/xenomorph/X = owner
+	if(X && !X.buckled && !X.is_mob_incapacitated())
+		return TRUE
```

Comment:
I don't think this override is needed as it's just duplicating things.

We check for `buckled` and `is_mob_incapacitated()` within `check_state()` which you call on line 52 and the base definition of the proc is enough.